actor-sdk/sdk-core-android/android-sdk/src/main/java/im/actor/sdk/controllers/activity/controllers/MainPhoneController.java

```diff
@@ -425,7 +432,7 @@ public void onResume() {
                     public void onChanged(Boolean isAppLoaded, Value<Boolean> Value,
                                           Boolean isAppEmpty, Value<Boolean> Value2) {
                         if (isAppEmpty) {
-                            if (isAppLoaded) {
+                            if (true || isAppLoaded) {
```

Comment:
oops, its a forgotten test! (recently we have some problems in syncing for new user, I temporarily skip the problem with this trick), I'll fix it.

anyway, whats about my approach?